Podfile

```
@@ -8,7 +8,9 @@ inhibit_all_warnings!
 use_frameworks! # Defaulting to use_frameworks! See pre_install hook below for static linking.
 use_modular_headers!
 
-platform :ios, '12.0'
+app_ios_dt = Gem::Version.new('12.0')
+
+platform :ios, app_ios_dt
```

Not sure the `platform` directive officially accepts a `Gem::Version` instead of a `String`? 🤔
(Maybe it does, but on phone rn so can't test it myself)

```suggestion
platform :ios, app_ios_dt.to_s
```

nit: also I realised that maybe we should pick better names for those variables than the one I chose when I suggested this solution 😅  e.g. `app_deployment_target`?

---

Thanks for taking a look @AliSoftware 

> Not sure the platform directive officially accepts a Gem::Version instead of a String? 🤔
It seemed to do so, but I updated to pass [.version ](https://ruby-doc.org/stdlib-2.5.0/libdoc/rubygems/rdoc/Gem/Version.html#method-i-version). 

> maybe we should pick better names
Renamed!